.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug",
+            "type": "gdb",
+            "request": "launch",
+            "target": "${workspaceFolder}/bazel-bin/<path to the bin file>",
+            "cwd": "${workspaceRoot}",
+            "valuesFormatting": "parseText"
+        }
+    ]
```

Comment:
Should we add the configuration for Windows too? Something like this should work.
```
        {
            "name": "Debug on Windows",
            "type": "cppvsdbg",
            "request": "launch",
            "program": "${workspaceFolder}/build/<path-to-bin-file>",
            "args": [],
            "stopAtEntry": false,
            "cwd": "${workspaceFolder}",
            "environment": [],
            "externalConsole": false
        }
```

Reply:
thanks, added